src/View/UIViewTrait.php

```diff
@@ -13,11 +13,16 @@
     /**
      * Initialization hook method.
      *
+     * ### Options:
+     *
+     * - `layout` to pass your layout, or it will use the plugin one
+     *
+     * @param array $options
      * @return void
      */
-    public function initializeUI()
+    public function initializeUI(array $options = [])
     {
-        if (empty($this->layout) || $this->layout === 'default') {
+        if (empty($options['layout'])) {
```

Comment:
I think better to allow one to pass `layout = false` or `layout = ...`  if they don't want the default one or if they want a custom one instead. If `TRUE` is passed or option is not defined, use plugin's default.

Right now, `FALSE` will still define the plugin's layout and `TRUE` will just do nothing.